fakegato-history.js

```diff
@@ -601,6 +655,19 @@ module.exports = function (pHomebridge) {
 					else
 						this._addEntry({ time: entry.time, temp: entry.temp, humidity: entry.humidity, pressure: entry.pressure });
 					break;
+				case TYPE_WEATHER2:
+					if (!this.disableTimer)
+						if(entry.temp !== undefined) {	// allow on / to be added to the data stream
+							homebridge.globalFakeGatoTimer.addData({ entry: entry, service: this });
+						} else {
+							this._addEntry({ time: entry.time, status: entry.status });
+						}
+					else
+						if(entry.power !== undefined) {	// allow on / to be added to the data stream
```

Comment:
are u sure, that you have to check the power property on an TYPE_WEATHER2 entry ?

Reply:
You are correct, I should be checking the 'temp' field. Will fix in the next push

Tks for reviewing